src/app/legacy/containers/SocialEmbed/index.test.jsx

```diff
@@ -39,17 +39,8 @@ describe('SocialEmbedContainer', () => {
       );
 
       expect(container.firstChild).toMatchSnapshot();
-      expect(
-        document.querySelector(
-          'head script[src="https://platform.twitter.com/widgets.js"]',
-        ),
-      ).toBeTruthy();
+      
```

Comment:
This deleted test here you can import fireEvent from `react-testing-library-with-providers` and do the same click invoking and it should work